### DISCUSSION TITLE
这个项目能不能支持同时导入很多人的数据，每个人的显示颜色不同？

### DISCUSSION THREAD

**Discussion by efish2002** [yihong0618/running_page#423] [Ideas]
我们有个跑步群，大家想把数据导到一起显示，应该可以实现吧。只是不同颜色显示有点麻烦

**Comment by yihong0618** [comment]
不可行，曾经想过。

现在特别不可行。自己的话没问题，多人的话，地理数据对国家来说现在属于敏感数据（参考 nike run club）下架。

如果跑步群自己做没有问题，从我这里来说没法承担相应的风险，抱歉。

这个先关了哈。


---

如果你想 fork 做我倒是可以提供技术帮助或思路。可以给我邮件。但是这个项目不会添加。

**Reply by efish2002** [reply]
好的，谢谢。只是一个想法，不够成熟，如果需要我跟您联系。
